src/components/FloatingDropdown/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const DATA_HOOKS = {
+  coreDropdown: 'core-dropdown',
+  base: 'floating-dropdown-base',
+};
+
+export const ICON_SIZE = 24;
```

Comment:
I would like to re-use it inside my st.css: https://github.com/wix/wix-ui-tpa/pull/313/files#diff-4c0e7e82a5365fefb909bd5e4b709f5eR23 - can I?

Reply:
Theoretically it might be able to import it (like we import the `formatters.st.js` file.
But this might cause a performance hit.
I guess for now it would be best to just have duplication :( 
Btw, from the brief review I gave, the component looks great!
I would consider using the `TextButton` instead of the regular `Button` 🤔

Reply:
Yes, you are right, I should use TextButton here :)

Will add it :)

Thanks! :)

Reply:
Hi, changed Base implementation to use TextButton: https://github.com/wix/wix-ui-tpa/pull/313/commits/4c7f002016dba5418f3c1dab9d34532787fa86cc